README.md

```diff
@@ -57,3 +57,83 @@ Releases are automatically pushed on commits merged to master that fulfill the f
 The release process is building all runtimes and then puts them into a release in this github repository.
 
 The format of [`CHANGELOG.md`](CHANGELOG.md) is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+# Weight Generation
+
+To generate weights for a runtime
+
+1. Build `chain-spec-generator` with `--profile production --features runtime-benchmarks`
+2. Use it to build a chain spec for your runtime, e.g. `./target/production/chain-spec-generator --raw polkadot-local > polkadot-chain-spec.json`
+3. Create `file_header.txt`
+
+```text
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+```
+
+4. `rsync` chain spec/s and the file header to a benchmark machine
+
+5. Build `polkadot-sdk` with `--profile production --features runtime-benchmarks` on the benchmark machine
+
+6. Create output directories for the weights on the benchmark machine
+
+7. Run on the benchmark machine:
+
+```bash
+for pallet in \
+  frame_system \
+  # other pallets you want to benchmark
```

Comment:
@liamaharon Maybe it's worth mentioning a call to list all pallets in the runtime, something like this (inspired by command-bot-scripts):
```
local pallets=($(
    ./target/production/polkadot benchmark pallet --list  |\
      --chain=/path/to/chain-spec.json |\
      tail -n+2 |\
      cut -d',' -f1 |\
      sort |\
      uniq
))

for pallet in "${pallets[@]}"; do
    echo "   [+] $pallet"
done
```

Reply:
Didn't know about that command, much better! Thanks 👌